src/VirtoCommerce.Platform.Modules/Local/LocalStorageModuleCatalog.cs

```diff
@@ -178,6 +181,37 @@ public override void Validate()
             return result;
         }
 
+        private void CopyAssembliesWithDistributedLock(IDictionary<string, ModuleManifest> manifests)
+        {
+            var storageLockResource = new DistributedLockResource(_redisConnMultiplexer, _options.DistributedLockWait, nameof(LocalStorageModuleCatalog));
+            storageLockResource.WithLock((x) =>
+            {
+                switch (x)
+                {
+                    // Delayed lock acquire, do nothing here with a notice logging
+                    case DistributedLockCondition.Delayed:
```

Comment:
could we move these logs to the DistibutedLockResourse with less payload-specific comments?